client/src/components/AboutPage/AboutHowSection/AboutHowSection.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FunctionComponent, RefObject } from 'react';
+
+// Assets
+import imgSearch from 'assets/images/search.png';
+import imgWidget from 'assets/images/widget.png';
+import imgThemeSwitch from 'assets/images/theme-switch.png';
+
+// Styles
+import { Container, Flex } from 'styles/global';
+import * as S from '../AboutPage.styles';
+
+const articles = [
+  {
+    title: 'Search your city',
+    content:
+      'Use search to find weather of your city. Just enter the name of the city in the searchbar and click on the preferred one in a dropdown window below.',
+    image: { src: imgSearch, alt: 'Example of using search' }
+  },
+  {
+    title: 'Look through different info types',
+    content:
+      'Check out different weather data by clicking through hourly, three-hourly and daily tabs. Click on the widget to expand it and see more detailed weather data. Use click-and-drag to view all the widgets.',
+    image: { src: imgWidget, alt: 'Example of using widget' }
+  },
+  {
+    title: 'Toggle light/dark mode',
+    content: 'Switch to light or dark mode and vice versa depending on your preference.',
+    image: { src: imgThemeSwitch, alt: 'Example of using theme-switch' }
+  }
+];
+
+interface AboutHowSectionProps {
+  customRef: RefObject<HTMLElement>;
+}
+
+const AboutHowSection: FunctionComponent<AboutHowSectionProps> = ({ customRef }) => (
```

Comment:
please rename `customRef`, not clear why it's custom and why it's needed according to current name

Reply:
At first, I named it as just `ref`, but got the following warnings in the console: "Warning: AboutHowSection: `ref` is not a prop. Trying to access it will result in `undefined` being returned. If you need to access the same value within the child component, you should pass it as a different prop. (https://reactjs.org/link/special-props)" and "react-dom.development.js:67 Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?"

Should I just write `aboutHowSectionRef={aboutHowSectionRef}`?

Reply:
if you won't use css, `aboutHowSectionRef` looks better than `customRef`